python/doc/usecases/use_case_flood_model.rst

```diff
@@ -97,6 +149,276 @@ We want to estimate the flood probability:
  .. math::
     P_f = P(S > 0).
```

Comment:
Use \Prob{S>0} instead of P(S>0).

Reply:
I just rebased on top of master, so that your PR #2392 was taken into account and fixes this. This is fixed then.